components/mapping-file/dead/index.mjs

```diff
@@ -0,0 +1,12 @@
+import { InternalAppmapError } from "../../error/index.mjs";
+import { generateDeadcode } from "../../util/index.mjs";
+
+export const fillSourceMap = generateDeadcode(
+  "forbidden call to fillSourceMap",
+  InternalAppmapError,
+);
+
+export const loadSourceMap = generateDeadcode(
+  "forbidden call to loadSourceMap",
+  InternalAppmapError,
+);
```

Comment:
This component instance is only enable for the browser execution environment. Instrumentation actually happens in the node execution environment (the top level component is server). However this is still broken because we only support `file://` urls. I haven't tested source mapping yet. This is next on my todo list. We probably want to fix this before release.

> This seems like a good way of getting file paths for JS files as well. Will this be supported?

I don't understand this :(

Reply:
Web servers can serve source maps in the same manner as they serve JavaScript if the source maps aren't inlined.

e.g.
```
GET http://localhost:3000/public/my-script.js
GET http://localhost:3000/public/my-script.js.map
```

From there we should be able to derive a file path on disk from looking at the `sources`.